.github/workflows/node-ci.yml

```diff
@@ -29,6 +29,12 @@ jobs:
               with:
                   fetch-depth: 0
 
+            - name: Force Node version
+              uses: actions/setup-node@v4
+              with:
+                  node-version: ${{ matrix.node-version }}
+                  cache: yarn
```

Comment:
**praise:** Looks good to me. I suspect the ubuntu images have a default version of Node installed. Being explicit with `setup-node` is smart.

Reply:
You're right.  Once we move to ubuntu 24 (which is in Beta) it'll default to Node 20. But I suspect we still want to define the Node version explicitly. 

https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md#language-and-runtime